data/json/items/corpses/inactive_bots.json

```diff
@@ -754,6 +754,61 @@
     "flags": [ "SINGLE_USE" ],
     "melee_damage": { "bash": 6, "cut": 6 }
   },
+  {
+    "type": "TOOL",
+    "id": "bot_yrax_triakis",
+    "symbol": ",",
+    "color": "light_gray",
+    "name": { "str_sp": "inactive triakis" },
+    "category": "other",
+    "description": "Gap in its shell reveals an interior made from a single piece of quartz, intricately carved into a series of nested tetrahedra.",
+    "price": "10 USD",
+    "material": [ "steel", "yrax_quartz" ],
+    "volume": "119 L",
+    "weight": "221 kg",
+    "to_hit": { "grip": "bad", "length": "hand", "surface": "any", "balance": "clumsy" },
+    "flags": [ "TRADER_AVOID", "NO_REPAIR", "SINGLE_USE" ],
+    "use_action": { "type": "effect_on_conditions", "menu_text": "Activate", "effect_on_conditions": [ "yrax_triakis_activation" ] },
+    "melee_damage": { "bash": 20, "cut": 15 }
+  },
+  {
+    "id": "bot_golden_monolith",
+    "type": "TOOL",
+    "symbol": "I",
+    "name": { "str": "inactive golden monolith" },
+    "description": "A large golden monolith.  Carefully carved asymmetric edges betray its artificial origin.",
+    "volume": "930 L",
+    "//": "A lot of gold in this bad boy.",
+    "weight": "8282 kg",
+    "price": "50 USD",
+    "price_postapoc": "50 USD",
+    "melee_damage": { "bash": 1000 },
```

Comment:
Moving this massive thing would be so slow I doubt you could damage anyone with it, even if they stand still (except maybe damaging their feet). (And unless it falls from a weight, but IIUC that is it's own thing, separate from me `melee_damage`.) So I think the `melee_damage` should be lower.

Reply:
I'm pretty sure i copied it from broken bot but Ill look into it.

Reply:
Fixed